configure.ac

```diff
@@ -94,7 +103,7 @@ if test -z "$O_VER"; then
   OBJCOPY=""
 fi
 if test -z "$OBJCOPY"; then
-  AC_PATH_PROG([LOBJCOPY], [llvm-objcopy])
+  AC_PATH_PROG([LOBJCOPY], [llvm-objcopy], [], [$PATH$PATH_SEPARATOR/usr/local/opt/llvm/bin])
```

Comment:
I think the better syntax in this case
would be `${PATH_SEPARATOR}`

Reply:
will do